### DISCUSSION TITLE
Underlig visning af Cph Opera Festival hos Kulturens Frivillige

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#21] [Bugs] [answered]
Er der en af jer, der kan se, hvad der går galt i visningen af Copenhagen Opera Festival på Kulturens Frivilliges portal? 

F.eks. "Crowdmanage og festlig stemning" vises fint. "Assistent på showbåden" gør ikke. 

https://rehelper.dk/portal/kulturens/event/copehagen-opera-festival-2wm0

![image](https://github.com/xooom-community/product/assets/167326849/6009fc6e-f96e-4939-853a-eeac66ac0f9f)

![image](https://github.com/xooom-community/product/assets/167326849/27d951a5-24e6-4ebe-860a-531d917b4f66)

**Comment by MartinQwist** [comment]
På siden Opgaver lader det til, at opgaven "Assistent på showbåden" kun bruges i 1 fremtidig opgave. Men af vagtplanen i backend og portal genbruges opgaven flere gange. Modsat med "Kor og playliste", som ikke bruges 26 gange. 

![image](https://github.com/xooom-community/product/assets/167326849/e44e274e-c2be-45f5-b24d-11f658e16495)

**Comment by mikkelsl** [comment, marked as answer]
Der er lavet en opgave på det her: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=73173204
